ToDoList/module/app/views/notes/NotesViewModel.swift

```diff
@@ -37,8 +40,15 @@ final class NotesViewModel {
         }
     }
     
+    func deleteButtonTapped() {
+        guard let indexPath = indexPath else { return }
+        delegate?.deleteItem(at: indexPath)
+        delegate?.popViewController()
+    }
+    
     func setupButtons() {
         delegate?.setupCancelButton(action: #selector(NotesViewController.cancelToggleButtonTapped(_:)))
         delegate?.setupSaveButton(action: #selector(NotesViewController.saveToggleButtonTapped(_:)))
+        delegate?.setupDeleteButton(action: #selector(NotesViewController.deleteToggleButtonTapped(_:)))
```

Comment:
pls rm to viewcontroller

Reply:
Setup button functions moves to view controller